roles/rust-foundation-project-director.md

```diff
@@ -0,0 +1,35 @@
+# Role Description: Rust Foundation Project Director
+
+Please refer to [Rust Foundation Director Roles & Responsibilities](TODO) for components of the role description that apply to all Foundation Board Directors.
+
+## Goals
+
+- Represent the needs and interests of the Rust Project at large within the Rust Foundation board
+- Ensure communication between the Rust Project and Foundation is robust and high fidelity.
+- TODO Fill out this list
+
+## Time Budget
+
+- 2-5 hours per week
+
+## Tasks / Activities / Responsibilities
+
+- [Abide by the Roles & Responsibilites of Foundation Board Members](https://foundation.rust-lang.org/static/board-director-role-description.pdf)
+- Attend Rust Foundation board meetings
+- Help prepare and present the "Project Director Update" for each board meeting
+- Meet with the Rust Foundation staff on a regular basis
+- Work with the Rust Leadership Council and Rust Foundation staff on refining the role of Project Director for the future
+- TODO Fill out this list
+
+## Qualities of a Council member (required skills)
+
+- Sufficient time and energy to dedicate to the needs of the Rust Foundation and the Rust Project.
+- An interest in helping with the topics of Project operations and Project governance.
+- Broad awareness of the needs of the Project outside of their teams or areas of active contribution.
+- A keen sense of the needs of Rust Project, its teams, processes, and ecosystem.
+- Ability to prioritize others' needs, representing all viewpoints from the Rust Project, not just those they agree with.
```

Comment:
Hmm, I think a director should be prepared to express their opinion on a matter as well (at least one that the council has not taken an official position on, but maybe even then). I think the role is more than just a liaison.

I also think the skill applies to more than just members of the project; it applies to other directors as well as foundation staff members and prominent community members.

This feels closer to the mark for me, though it could be improved..

```suggestion
- Willingness to empathize with, and express, views that they may not have personal experience with or necessarily agree with.
```

Reply:
yea I'm not feeling thrilled with the wording but I can't figure out how to describe my objection, it just feels like pure vibes, like preferring the word "viewpoints" to "views" and "understand" over "empathize with", but I think those are ultimately coming from my viewpoint, lol, and the things these words mean to me. How "viewpoints" is centering people rather than ideas, and how "understanding" is centering the goal of representation, whereas "empathizing" to me sounds to me like it could boil down to "so and so is upset about this" without really understanding _why_ they're upset.

In retrospect that actually wasn't that hard to figure out... lol

```suggestion
- Willingness to understand and represent viewpoints they may not have personal experience with or necessarily agree with.
```

Reply:
I noticed in the reword that we lost "from the Rust project." Do we want to keep that?

Reply:
I'm happy with the wording @yaahc proposed.

> I noticed in the reword that we lost "from the Rust project." Do we want to keep that?

That was deliberate on my part:

> I also think the skill applies to more than just members of the project; it applies to other directors as well as foundation staff members and prominent community members.